scalameta/src/main/scala/scala/meta/internal/parsers/Parsers.scala

```diff
@@ -1715,6 +1726,10 @@ private[meta] class Parser(val input: Input)(implicit val dialect: Dialect) { pa
 
   def enumerator(isFirst: Boolean, allowNestedIf: Boolean = true): List[Enumerator] =
     if (token.is[`if`] && !isFirst) autoPos(Enumerator.Guard(guard().get)) :: Nil
+    else if (token.is[Ellipsis]) {
+      next()
+      constructR1Quasi[Enumerator.Generator](expr())
+    }
```

Comment:
It does not. However, is there any case where such functionality would be useful? `blah` = `blahs.last`. 
I think it's not so hard to add it, if necessary.